.github/workflows/ci.yml

```diff
@@ -0,0 +1,129 @@
+name: windows_exporter CI/CD
+  
+# Trigger on pull requests and releases
+# Deployments will only occur for releases (see `if` clauses in the build job).
+on:
+  pull_request:
+    branches:
+      - master
+  release:
+    types:
+      - published
+      - edited
+
+jobs:
+  test:
+    runs-on: windows-2019
```

Comment:
As I understand from [the runner docs](https://github.com/actions/virtual-environments/blob/main/images/win/Windows2019-Readme.md#go), this image will default us to Go 1.15. The AppVeyor image we've been using is (currently, at least) on 1.17.
It looks like we can opt in to running a more modern version by tweaking so we use `GOROOT_1_17_X64`? Or is there more to it?

Reply:
It looks like [setup-go](https://github.com/actions/setup-go) is the reccommended way to use the 1.17 installation.

I've added the action to the CI file in a recent push.